src/main/java/org/datadog/jmxfetch/AppConfig.java

```diff
@@ -122,6 +122,12 @@ public class AppConfig {
             required = false)
     private boolean statsdTelemetry;
 
+    @Parameter(
+            names = {"--instance_telemetry", "-it"},
```

Comment:
```suggestion
            names = {"--jmxfetch_telemetry", "-it"},
```
or maybe `internal_telemetry`? or maybe just `telemetry`?

In the agent we simply use `telemetry`, so maybe that works for us here.